.gitignore

```diff
@@ -8,3 +8,4 @@ private
 *.pyc
 .DS_Store
 sluggoAPI/**
+env/**
```

Comment:
If we are doing this we should remove sluggoAPI from the gitignore and update the readme. I think standardizing to env or venv is good.

Reply:
Do what you want.

Reply:
can be adjusted another time, not worth waiting, I say 🚢